src/pages/journal/journal-components/journal-candidate/__tests__/journal-candidate.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
In my opinion there is not too much need for the components inside of `journal-components` to be marked with a `journal-` prefix, because we will know it for sure that these components are part of `journal` when we look at the folder structure. It'll act like a namespace.